nca/NetworkConfig/LiveSim/dns_pods.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-system
+  labels:
+    name: kube-system
+---
+apiVersion: apps/v1
+kind: Pod
+metadata:
+  name: core-dns-livesim
+  namespace: kube-system
+  labels:
+    app: core-dns
```

Comment:
The actual label for this pod is `k8s-app: kube-dns`

Reply:
Would also consider adding the label: `projectcalico.org/namespace: kube-system`

Reply:
done.